nuxeo-core/nuxeo-core-convert-plugins/src/main/resources/OSGI-INF/convert-service-contrib.xml

```diff
@@ -108,6 +108,33 @@
       <destinationMimeType>text/plain</destinationMimeType>
     </converter>
 
+    <converter class="org.nuxeo.ecm.platform.convert.plugins.CommandLineConverter" name="ai2pdf">
+      <sourceMimeType>application/postscript</sourceMimeType>
+      <sourceMimeType>application/eps</sourceMimeType>
+      <sourceMimeType>application/x-eps</sourceMimeType>
+      <sourceMimeType>image/eps</sourceMimeType>
+      <sourceMimeType>image/x-eps</sourceMimeType>
+      <sourceMimeType>application/illustrator</sourceMimeType>
+      <destinationMimeType>application/pdf</destinationMimeType>
```

Comment:
There's still the converter above which is named `ai2pdf`. And the `CommandLineName` below.